app/views/application/_toast.html.erb

```diff
@@ -1,6 +1,39 @@
-<% if notice %>
-  <p class="alert alert-success"><%= notice %></p>
-<% end %>
-<% if alert %>
-  <p class="alert alert-danger"><%= alert %></p>
-<% end %>
+<div x-data="{ show: false }"
+  x-show="show"
+  x-init="setTimeout(() => { show = true }, 0); setTimeout(() => { show = false }, 5000)"
+  x-transition:enter="transition ease-out duration-300"
+  x-transition:enter-start="opacity-0 transform scale-90"
+  x-transition:enter-end="opacity-100 transform scale-100"
+  x-transition:leave="transition ease-in duration-300"
+  x-transition:leave-start="opacity-100 transform scale-100"
+  x-transition:leave-end="opacity-0 transform scale-90"
+  class="flex justify-center">
+  <div class="<%= bg_color %> flex rounded-md p-4 max-w-sm">
+    <div class="flex-shrink-0 <%= icon_color %>">
+      <%= render icon, w: 'w-5', h: 'h-5' %>
+    </div>
+    <div class="ml-3 text-sm font-medium <%= text_color %>">
+      <% if message.kind_of?(Array) %>
+          <% if message.size == 1 %>
+            <p><%= message.first %></p>
```

Comment:
I would simplify this to always do one of the rendering styles. A subtle frustrating UX for someone managing this in the future.

Reply:
Going to leave this since I have active use cases that show multiple errors and others that show a single message.